src/features/session_replay/instrument/index.js

```diff
@@ -9,30 +9,45 @@
  * It is not production ready, and is not intended to be imported or implemented in any build of the browser agent until
  * functionality is validated and a full user experience is curated.
  */
-import { MODE } from '../../../common/session/constants'
+import { DEFAULT_KEY, MODE, PREFIX } from '../../../common/session/constants'
 import { InstrumentBase } from '../../utils/instrument-base'
 import { FEATURE_NAME } from '../constants'
+import { isPreloadAllowed } from '../shared/utils'
 
 export class Instrument extends InstrumentBase {
   static featureName = FEATURE_NAME
   constructor (agentIdentifier, aggregator, auto = true) {
     super(agentIdentifier, aggregator, FEATURE_NAME, auto)
+    let session
     try {
-      const session = JSON.parse(localStorage.getItem('NRBA_SESSION'))
-      if (session.sessionReplayMode !== MODE.OFF) {
-        this.#startRecording(session.sessionReplayMode)
-      } else {
-        this.importAggregator({})
-      }
-    } catch (err) {
-      this.importAggregator({})
+      session = JSON.parse(localStorage.getItem(`${PREFIX}_${DEFAULT_KEY}`))
```

Comment:
We talked about changing this to just use the session entity class yesterday. If you want, we can move this PR forward as-is and I can do the work to move to using session entity class in a separate PR.

Reply:
yeah that'd probably be for the best